tests/platform_tests/api/test_sfp.py

```diff
@@ -525,9 +525,13 @@ def test_tx_disable_channel(self, duthosts, enum_rand_one_per_hwsku_hostname, lo
                 logger.warning("test_tx_disable_channel: Skipping transceiver {} (not applicable for this transceiver type)".format(i))
                 continue
 
+            orig_tx_disable_chan_mask = sfp.get_tx_disable_channel(platform_api_conn, i)
+            logger.info("Before disabling tx_channels, original tx channel masks is {}".format(orig_tx_disable_chan_mask))
+
             # Test all TX disable combinations for a four-channel transceiver (i.e., 0x0 through 0xF)
             # We iterate in reverse here so that we end with 0x0 (no channels disabled)
             for expected_mask in range(0xF, 0x0, -1):
```

Comment:
can we handle QSFP-DD where the number of lanes can be up to 8, current test goes up to 4 only

Reply:
Done in latest commit